### ISSUE TITLE
Not compatible with latest Babel

### ISSUE THREAD

**Issue by lexkrstn** [lingui/js-lingui#211]
After updating my project's Babel version up to 7.0.0-beta.46 I began getting the following error from the  `lingui extract` command (installed globally):

> Error: [BABEL] %MyProjectPath%: Requires Babel "^7.0.0-beta.41", but was loaded with "7.0.0-beta.40". You'll need to update your @babel/core version. (While processing: "%MyProjectPath%\\node_modules\\@babel\\plugin-proposal-object-rest-spread\\lib\\index.js")

`%MyProjectPath%` here is the placeholder I replaced the project path with to shorten the text.

Babel version: 7.0.0-beta.46

**Comment by kafein** [comment]
try with npm install -g @lingui/cli@2.0.8

**Comment by alesn** [comment]
It might be worth putting a note in the Readme saying something like: "If you use Babel 7, use lingui version 2.0.8 until lingui 3 is released" because a large number of projects is running on Babel 7 (though it is still in "beta") and can't use lingui unless it runs with Babel 7

**Comment by tricoder42** [comment]
@alesn Good point! `master` still uses Babel 7 so it might be worth to release 3.0.0-beta... I need to update release pipeline to support it though.
